routing_common/transit_types.cpp

```diff
@@ -125,7 +125,7 @@ bool SingleMwmSegment::IsValid() const
 // Gate -------------------------------------------------------------------------------------------
 Gate::Gate(FeatureId featureId, bool entrance, bool exit, double weight,
            std::vector<StopId> const & stopIds, m2::PointD const & point)
-  : m_featureId(featureId)
+  : m_osmId(featureId)
```

Comment:
Do we need to fix param name too?

Reply:
I guess, yes. In that case we have param name and field name the same: `DECLARE_VISITOR_AND_DEBUG_PRINT(Gate, visitor(m_osmId, "osm_id"), ...`

Reply:
Not only name, I guess. But the type of param as well.

Reply:
I'm adding a commit about that

Reply:
Don